locales/ko.json

```diff
@@ -2,734 +2,943 @@
     },
     "aria-required-attr": {
-      "description": "ARIA 역할을 가진 요소가 모든 필수 ARIA 속성을 갖도록 합니다.",
-      "help": "필수 ARIA 속성이 제공 되어야 합니다."
+      "pass": "모든 필수 ARIA 어트리뷰트가 존재합니다.",
+      "fail": {
+        "singular": "필수 ARIA 어트리뷰트가 없습니다: ${data.values}",
+        "plural": "필수 ARIA 어트리뷰트들이 없습니다: ${data.values}"
+      }
     },
     "aria-required-children": {
-      "pass": "필수적인 ARIA 역할이 존재합니다.",
-      "fail": "필수적인 ARIA 역할이 제공되지 않고 있습니다:  ${data.values}",
-      "incomplete": "ARIA의 자손 역할이 추가되어야 합니다:  ${data.values}"
+      "pass": "필수 ARIA 하위 항목들이 존재합니다.",
+      "fail": {
+        "singular": "필수 ARIA 하위 역할(role)이 없습니다: ${data.values}",
+        "plural": "필수 ARIA 하위 역할(role)들이 없습니다: ${data.values}"
+      },
+      "incomplete": {
+        "singular": "추가 될 예상 ARIA 하위 역할(role): ${data.values}",
+        "plural": "추가 될 예상 ARIA 하위 역할(role)들: ${data.values}"
+      }
     },
     "aria-required-parent": {
-      "pass": "필수적인 ARIA 역할이 존재합니다.",
-      "fail": "필수적인 ARIA 역할이 제공되지 않고 있습니다:  ${data.values}"
+      "pass": "필수 ARIA 상위 역할(role)이 존재합니다.",
+      "fail": {
+        "singular": "필수 ARIA 상위 역할(role)이 없습니다: ${data.values}",
+        "plural": "필수 ARIA 상위 역할(role)들이 없습니다: ${data.values}"
+      }
     },
-    "aria-unsupported-attr": {
-      "pass": "사용된 ARIA 속성은 지원되고 있습니다.",
-      "fail": "사용된 ARIA 속성은 스크린 리더나 보조 기술에 널리 지원되지 않고 있습니다:  ${data.values}"
+    "aria-roledescription": {
+      "pass": "aria-roledescription이 지원되는 의미론 역할(role)에 사용되었습니다.",
+      "incomplete": "지원되는 스크린리더에서 aria-roledescription이 낭독되는지 검사하세요.",
+      "fail": "엘리먼트에 aria-roledescription을 지원하는 역할(role)을 제공하세요."
     },
-    "unsupportedrole": {
-      "pass": "사용된 ARIA 역할이 지원됩니다.",
-      "fail": "사용된 ARIA 속성은 스크린 리더나 보조 기술에 널리 지원되지 않고 있습니다.:  ${data.values}"
+    "aria-unsupported-attr": {
+      "pass": "ARIA 어트리뷰트가 지원됩니다.",
+      "fail": "ARIA 어트리뷰트가 스크린리더와 보조 기술에서 널리 지원되지 않습니다: ${data.values}"
     },
     "aria-valid-attr-value": {
-      "pass": "ARIA 속성이 유효합니다.",
-      "fail": "무효인 ARIA 속성입니다: ${data.values}",
-      "incomplete": "페이지에 ARIA 속성의 요소 ID가 존재하지 않습니다: ${data.values}"
+      "pass": "ARIA 어트리뷰트 값이 유효합니다.",
+      "fail": {
+        "singular": "유효하지 않은 ARIA 어트리뷰트 값: ${data.values}",
+        "plural": "유효하지 않은  ARIA 어트리뷰트 값들: ${data.values}"
+      },
+      "incomplete": {
+        "noId": "ARIA 어트리뷰트 엘리먼트 ID가 페이지에 없습니다: ${data.needsReview}",
+        "ariaCurrent": "ARIA 어트리뷰트 값이 유효하지 않으며 \"aria-current=true\"로 취급 될 것입니다: ${data.needsReview}"
+      }
     },
     "aria-valid-attr": {
-      "pass": "ARIA 속성명이 유효합니다.",
-      "fail": "무효인 ARIA 속성명 입니다:  ${data.values}"
+      "pass": "ARIA 어트리뷰트 이름이 유효합니다.",
+      "fail": {
+        "singular": "유효하지 않은 ARIA 어트리뷰트 이름: ${data.values}",
+        "plural": "유효하지 않은 ARIA 어트리뷰트 이름들: ${data.values}"
+      }
+    },
+    "fallbackrole": {
+      "pass": "하나의 역할(role) 값만 사용되었습니다.",
+      "fail": "폴백 역할(role)들은 구형 브라우저들에서 지원되지 않으므로, 단 하나의 값만 사용하세요.",
```

Comment:
사용하세요. -> 사용해야 합니다

Reply:
@WilcoFiers

The original text is described as "Use only role 'presentation' or 'none' since they are synonymous."
And in Korean, "~야 합니다" is the should or must of English.
Is should or must meaning right?

Reply:
"must" would be more appropriate.

Reply:
I revised it as you suggested.